code/game/objects/effects/decals/crayon.dm

```diff
@@ -31,4 +32,45 @@
 
 	add_overlay(list(mainOverlay, shadeOverlay))
 
+	SSicon_cache.crayon_cache.Cut()
+
 	add_hiddenprint(usr)
+
+/obj/effect/decal/cleanable/chalk
+	name = "rune"
+	desc = "A strange collection of symbols drawn in chalk."
+	icon = 'icons/obj/rune.dmi'
+	layer = 2.1
+	anchored = 1
+	smooth = SMOOTH_TRUE	
+
+/obj/effect/decal/cleanable/chalk/Initialize(mapload, main = "#FFFFFF", shade = "#000000", var/type = "rune")
+	. = ..()
+
+	name = type
```

Comment:
this variable name is very confusing because type is also a completely different variable on atoms

Reply:
Copy-pasted from crayon code.  I've changed all instances to drawtype to match the initializing arguments in crayons.dm.